src/common/payment.ts

```diff
@@ -156,6 +165,47 @@ export class TurboUnauthenticatedPaymentService
   ): Promise<TurboCheckoutSessionResponse> {
     return this.getCheckout(params);
   }
+
+  public async submitFundTransaction(
+    tx_id: string,
+  ): Promise<Omit<TurboCryptoFundResponse, 'target' | 'reward'>> {
+    const response = await this.httpService.post<TurboPostBalanceResponse>({
+      endpoint: `/account/balance/${this.token}`,
+      data: Buffer.from(JSON.stringify({ tx_id })),
+    });
+
+    if ('creditedTransaction' in response) {
+      return {
+        id: response.creditedTransaction.transactionId,
+        quantity: response.creditedTransaction.transactionQuantity,
+        owner: response.creditedTransaction.destinationAddress,
+        winc: response.creditedTransaction.winstonCreditAmount,
+        token: response.creditedTransaction.tokenType,
+        status: 'confirmed',
+        block: response.creditedTransaction.blockHeight,
+      };
+    } else if ('pendingTransaction' in response) {
+      return {
+        id: response.pendingTransaction.transactionId,
+        quantity: response.pendingTransaction.transactionQuantity,
+        owner: response.pendingTransaction.destinationAddress,
+        winc: response.pendingTransaction.winstonCreditAmount,
+        token: response.pendingTransaction.tokenType,
+        status: 'pending',
+      };
+    } else if ('failedTransaction' in response) {
+      return {
+        id: response.failedTransaction.transactionId,
+        quantity: response.failedTransaction.transactionQuantity,
+        owner: response.failedTransaction.destinationAddress,
+        winc: response.failedTransaction.winstonCreditAmount,
+        token: response.failedTransaction.tokenType,
+        status: 'failed',
+      };
+    }
+
+    throw new Error('Unknown response from payment service: ' + response);
```

Comment:
nit: use `switch` statement with default behavior to throw

Reply:
tried this but I get type errors on build 
![image](https://github.com/ardriveapp/turbo-sdk/assets/60207215/37bb9266-d1d1-4798-8e1a-f2d4119fe421)